boards/acd52832/doc.txt

```diff
@@ -0,0 +1,5 @@
+/**
+ * @defgroup    boards_acd52832 ACD52832
+ * @ingroup     boards
+ * @brief       Support for the aconno™ ACD52832
```

Comment:
Are we okay with Unicode characters?

Reply:
in #8379  this wasn't a problem, there's also one in cc2650stk doc

Reply:
I would remove such anyway (or replace by `(tm)` if necessary), I see super- and sub-scripts rendered badly more often than not.

Reply:
I would also try to remove Unicode, same reason as @smlng says. In fact, it has happened to us in a RIOT related project